psamm/datasource/sbml.py

```diff
@@ -701,14 +702,20 @@ def _get_flux_bounds(self, r_id, model, flux_limits, equation):
             else:
                 lower_str = text_type(-model.get_default_flux_limit())
         else:
-            lower_str = text_type(flux_limits[r_id][0])
+            lower = flux_limits[r_id][0]
+            if lower % 1 == 0 and lower != -model.get_default_flux_limit():
```

Comment:
I don't get the second condition (`lower != -model.get_default_flux_limit()`)... why only convert to `int` if it is not at the limit?